README.md

```diff
@@ -89,6 +89,21 @@ For advanced usage options, try
 tinysearch --help
 ```
 
+Please check what's required to [host WebAssembly in production](https://rustwasm.github.io/book/reference/deploying-to-production.html) -- you will need to explicitly set mime gzip types
+## Docker
+
```

Comment:
Maybe we can have a `run` command as well.
Something like `docker run -v `pwd`/index.json:/index.json tinysearch/tinysearch /index.json`
That's entirely too many `index.json`'s, but I don't know of a better way. 😆